core/src/test/scala/com/databricks/labs/remorph/transpilers/SnowflakeToDatabricksTranspilerTest.scala

```diff
@@ -263,6 +263,30 @@ class SnowflakeToDatabricksTranspilerTest extends AnyWordSpec with TranspilerTes
         "SELECT car_model, COUNT(DISTINCT city) FROM dealer GROUP BY ALL;"
     }
 
+    "transpile LCA replacing aliases" in {
+      "SELECT column_a as alias_a FROM table_a where alias_a = '123';" transpilesTo
+        "SELECT column_a as alias_a FROM table_a where column_a = '123';"
+    }
+
+    "transpile LCA replacing aliased literals" in {
+      "SELECT '123' as alias_a FROM table_a where alias_a = '123';" transpilesTo
+        "SELECT '123' as alias_a FROM table_a where '123' = '123';"
+    }
+
+    "transpile LCA with aliased table" in {
+      "SELECT t.col1, t.col2, t.col3 AS ca FROM table1 t WHERE ca in ('v1', 'v2');" transpilesTo
+        "SELECT t.col1, t.col2, t.col3 AS ca FROM table1 as t WHERE t.col3 in ('v1', 'v2');"
+    }
+
+    "transpile LCA with partition" in {
+      "SELECT t.col1 AS ca, ROW_NUMBER() OVER (PARTITION by ca ORDER BY ca) FROM table1 t;" transpilesTo
+        "SELECT t.col1 AS ca, ROW_NUMBER() OVER (PARTITION by t.col1 ORDER BY t.col1 ASC NULLS LAST) FROM table1 AS t;"
```

Comment:
Please add a comment for this explaining that although LCAs are supported, LCAS within a window expression is not supported (and is therefore subject to dealiasing).

Along this line of thought I'd also like to see a test showing that normal LCAs are left alone (and not dealiased).

Reply:
There is a comment about that in the implementation of `DealiasLCAs`